terraform/modules/gost_postgres/variables.tf

```diff
@@ -25,15 +25,9 @@ variable "subnet_ids" {
   type        = list(string)
 }
 
-variable "allowed_security_groups" {
-  description = "Security Group IDs which should be allowed to connect (ingress) to Postgres."
-  type        = list(string)
-}
-
-variable "allowed_cidr_blocks" {
-  description = "CIDR blocks which should be allowed to connect (ingress) to Postgres."
-  type        = list(string)
-  default     = []
```

Comment:
Removed this because we aren't using it – all our ingress rules are based around security groups.